### PR TITLE
17636 - Refactoring of AuthorizationRequestSentDialog, further unit testing

### PR DESCRIPTION
**Pull request by rodrigo-barraza**
*Issue #:*
https://github.com/bcgov/entity/issues/17636

*Description of changes:17636*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Comment by seeker25** [comment]
From the other ticket:

**Comment by seeker25** [comment]
Still missing the changes in the screenshots?

**Review by seeker25 — CHANGES_REQUESTED** [review]
Missing Ethan's UX changes

**Comment by seeker25** [comment]
image

**Comment by seeker25** [comment]
Ticket number not in the title

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2536-zrt9q6wv.web.app

**Comment by seeker25** [comment]
Looks good
image

File: auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue [hunk 1]
```diff
@@ -95,17 +95,22 @@
...
-                x-small
-                :color="getAffiliationInvitationStatus(item.affiliationInvites) === AffiliationInvitationStatus.Expired
-                  ? 'red' : 'primary'"
-              >
-                {{ getAffiliationInvitationStatus(item.affiliationInvites) === AffiliationInvitationStatus.Expired
-                  ? 'mdi-alert' : 'mdi-account-cog' }}
-              </v-icon>
+            <v-icon
+              class="pr-1 status-icon"
+              style="margin-top: 2px;"
```
Comment (seeker25) [inline auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue:102]: Duplicated by styling

**Review by Jxio — APPROVED** [review]

**Pull request merged**